ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -65,6 +65,20 @@ open class TabBarItem: NSObject {
                   accessibilityLabelBadgeFormatString: nil)
     }
 
+    /// Initializes `TabBarItem` for inter-item spacing.
+    /// This tab bar item does not have an image or title, it is used to add space between two other tab bar items.
+    @objc public init(emptySpace: CGFloat) {
```

Comment:
seems weird that empty space is the only param in initializer

Reply:
Can you elaborate on that?

Reply:
so, why aren't we taking a similar approach like UIStackView where you can define the padding per item rather than creating an empty item? initializing tabbaritem with empty space value seems bit weird to me.

Reply:
We can also take this approach too. Take a look at the latest iteration.